src/hooks/useQueryLogs.ts

```diff
@@ -88,24 +86,24 @@ export const useQueryLogs = () => {
 			setError(null);
 
 			const logsQueryRes = isQuerySearchActive
-				? await getQueryResult({ ...logsQuery, access: [] }, appendOffsetToQuery(custSearchQuery, logsQuery.pageOffset))
-				: await getQueryLogs(logsQuery);
+				? await getQueryResult({ ...logsQuery, access: [] }, appendOffsetToQuery(custSearchQuery, logsQuery.pageOffset),{params:"fields=true"})
+				: await getQueryLogs(logsQuery, {params:"fields=true"});
```

Comment:
You should send {fields: true} to a parser and that parser should parse it to a query string.
This parser should be in URL constants.

Reply:
How about an array containing the params?

Reply:
so that we can chain params if we need to do that in future?

Reply:
instead of boolean I think string will be better suited in this case.
{fields:  "true"}

Reply:
expectation: 

parser(object) => queryString

Reply:
it will fullfill this purpose